sessions/session-c.md

```diff
@@ -0,0 +1,48 @@
+# Session C: Meshing with cjdns and messing with secure-scuttlebutt
```

Comment:
Dig this! But I think we need a short title (the second could be like the after-colon part?)
e.g. "Build a Mesh Node: Meshing with cjdns and messing with secure-scuttlebutt" and I'll use the short one on the website schedule?